support-files/mysql.server.sh

```diff
@@ -53,6 +53,8 @@ datadir=
 # Negative numbers mean to wait indefinitely
 service_startup_timeout=900
 
+[ -e /etc/sysconfig/mysql ] && . /etc/sysconfig/mysql
```

Comment:
It is for centos only? what about debian/ubuntu?

Reply:
This change is only for centos. 
DEB has `[ -e /etc/default/mysql ] && . /etc/default/mysql || true` in init.d file